dualscreeninfo/src/dualscreeninfo/index.ts

```diff
@@ -31,17 +32,20 @@ interface IDualScreenInfoModule extends ExposedNativeMethods {
 	hingeWidth: number;
 	isSpanning: boolean;
 	windowRects: WindowRect[];
+	rotation: DeviceRotation;
 }
 
 class RNDualScreenInfoModule implements IDualScreenInfoModule {
 	private mIsSpanning: boolean = false;
 	private mWindowRects: WindowRect[] = [];
+	private mRotation: DeviceRotation = 'rotation0';
```

Comment:
Again should not need to hard-code "rotation0"

Reply:
Its ok since we have typecheck by Typescript